README.md

```diff
@@ -1,64 +1,5 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/{APP-NAME}-app.svg?style=shield)](https://circleci.com/gh/giantswarm/{APP-NAME}-app)
+[![CircleCI](https://circleci.com/gh/giantswarm/cluster-api-app.svg?style=shield)](https://circleci.com/gh/giantswarm/cluster-api-app)
 
-# {APP-NAME} chart
+# cluster-api chart
 
-Giant Swarm offers a {APP-NAME} App which can be installed in workload clusters.
```

Comment:
This ain't workload cluster app but the control plane for Cluster API management cluster components.